jekyll/_cci2/github-apps-integration.adoc

```diff
@@ -0,0 +1,255 @@
+---
+contentTags:
+  platform:
+  - Cloud
+---
+= GitHub App integration overview
+:page-layout: classic-docs
+:page-liquid:
+:page-description: Learn how to integrate GitHub with CircleCI to manage your GitHub CI/CD pipelines.
+:icons: font
+:experimental:
+
+[NOTE]
+====
+**GitHub authentication with CircleCI is changing**. Starting 14th August 2023 when you authenticate your CircleCI account with GitHub, this will be done through our GitHub App, rather than the GitHub OAuth app. You can see which account type you have by heading to the CircleCI web app and inspecting the URL in your browser:
```

Comment:
There's going to be a rollout period so only some users will be on GitHub App starting that week. Don't know if we want to clarify that it's being rolled out as opposed to all users getting it from 8/14.